students/uss_tomasz/lesson_14_decorators/14_1/sort_decorator.py

```diff
@@ -0,0 +1,14 @@
+def sort_decorator(func):
+    def inner_function():
+        temp = func()
+        temp.sort()
```

Comment:
It's safer to create new copy of sorted list instead of sorting in-place. Especially when creating a utility that will operate on various functions.

Imagine the case:
```
GLOBAL_CONST = [4, 2, 1, 3]

@sort_decorator
def data_feeder():
    return GLOBAL_CONST

print("GLOBAL_CONST = ", GLOBAL_CONST)
print("Sorted feeder = ", data_feeder())
print("GLOBAL_CONST = ", GLOBAL_CONST)
```

using `sorted` instead of `array.sort` would fix the mutation problem
https://discuss.codecademy.com/t/what-is-the-difference-between-sort-and-sorted/349679